src/themes/eui-amsterdam/overrides/_index.scss

```
@@ -1,10 +1,11 @@
 @import 'button';
 @import 'button_empty';
 @import 'button_group';
+@import 'call_out';
 @import 'flyout';
 @import 'header';
 @import 'image';
 @import 'modal';
 @import 'overlay_mask';
 @import 'popover';
-@import 'text';
+@import 'text';
```

nit: Space at the end of doc
```suggestion
@import 'text';

```